draft-ietf-tls-tls13.md

```diff
@@ -3311,10 +3413,45 @@ because TLS does not directly use this secret for anything
 other than for computing other secrets.)
 
 
-#  Mandatory Cipher Suites
+#  Mandatory Algorithms
+
+##  MTI Cipher Suites
+
+In the absence of an application profile standard specifying otherwise, a
+TLS-compliant application MUST implement the cipher suite
+[TODO:Needs to be selected](https://github.com/tlswg/tls13-spec/issues/32).
+(See {{cipher-suites}} for the definition.)
+
+##  MTI Extensions
 
 In the absence of an application profile standard specifying otherwise, a
-TLS-compliant application MUST implement the cipher suite [TODO:Needs to be selected](https://github.com/tlswg/tls13-spec/issues/32). (See {{cipher-suites}} for the definition.)
+TLS-compliant application MUST implement the following TLS extensions:
+
+  * Signature Algorithms ("signature_algorithms"; {{signature-algorithms}})
+  * Negotiated Groups ("supported_groups"; {{negotiated-groups}})
+  * Client Key Share ("client_key_share"; {{client-key-share}})
+  * Pre-Shared Key Extension ("pre_shared_key"; {{pre-shared-key-extension}})
+  * Server Name Indication ("server_name"; Section 3 of {{RFC6066}})
+
+All implementations MUST use the "signature_algorithms" extension when
+offering and negotiating certificate authenticated cipher suites.
+All implementations MUST use the "supported_groups" and "client_key_share"
+extensions when offering and negotiating DHE or ECDHE cipher suites.
+All implementations MUST use the "pre_shared_key" extension when
+offering and negotiating PSK cipher suites.
+All implementations MUST support use of the "server_name" extension
+with applications capable of using it.
```

Comment:
The "signature_algorithms" and "supported_groups" sections have requirements to fatal alert without them. The "server_name" extension error is specified just below here. "client_key_share" and "pre_shared_key" don't have explicit error expectations other than guaranteed failure without them.